### ISSUE TITLE
CarPlay safe mode issue

### ISSUE THREAD

**Issue by L0wnWolf** [AnemoneTeam/Anemone#297]
I’m using an iPhone X running iOS 11.3.1 with the Electra MPTCP jailbreak. Every time I plug my phone into my car for CarPlay, it resprings and goes into safe mode. I looked at the springboard crash logs, and AnemoneIcons.dylib is listed. Upon disabling AnemoneIcons.dylib, CarPlay works 100% fine without my phone respringing and going into safe mode. However, this turns the theme off for all my icons which is not ideal. Does anyone have a fix for this without having to disable AnemoneIcons.dylib? I have tried disabling the HTML and Fonts for Anemone, but neither of those fixed the issue.

**Comment by Root3287** [comment]
Yeah mine just crashes if I disconnect from CarPlay and go to the homescreen. This been happening since Yalu jailbreak...
